incubator/kafka/templates/statefulset.yaml

```diff
@@ -131,10 +131,10 @@ spec:
         - name: JMX_PORT
           value: "{{ .Values.jmx.port }}"
         {{- end }}
-        - name: POD_NAME
+        - name: POD_IP
```

Comment:
Is there a reason we are replacing `POD_NAME` with `POD_IP` instead of just adding it?

Reply:
Because there are 2 duplicate POD_NAMEs:
* https://github.com/helm/charts/blob/master/incubator/kafka/templates/statefulset.yaml#L134
* https://github.com/helm/charts/blob/master/incubator/kafka/templates/statefulset.yaml#L138

Reply:
haha yea, thats a pretty good reason :). Thanks for pointing it out. 

/lgtm